addons/message_paid_research/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api_version": "0.1",
+  "id": "message_paid_research",
+  "name": "Participate in paid research to help make VPN better",
+  "type": "message",
+  "translatable": false,
+  "conditions": {
+    "min_client_version": "2.15.0",
+    "translation_threshold": 0,
```

Comment:
is `translation_threshold` a 2.15+ feature? Is that why it's only on this message?
Also, are there any other differences between the messages besides `translation_threshold` and min/max versions?

Reply:
the 2.14 version will still get released, regardless of translation status as well, correct?

Reply:
translation_threshold is indeed only 2.15 and is the reason why there are two addOns for essentially the same thing. There shouldn't be any content differences.

And yes, both should go live and need no translation as they should both be targeting only the 'en' locale. Country disambiguation is happening on the survey itself.